src/onegov/form/parser/grammar.py

```diff
@@ -306,6 +306,18 @@ def url() -> ParserElement:
     return Suppress(Regex(r'https?://')).setParseAction(tag(type='url'))
 
 
+def video_url() -> ParserElement:
+    """ Returns an video url field parser.
+
+    Example::
+
+        vimeo-youtube
+
+    """
+    return Suppress(Regex(r'vimeo-youtube')).setParseAction(tag(
```

Comment:
I'm not crazy about the syntax for this field, but I also don't really have a better suggestion. Maybe ask around if someone has a good idea.

Reply:
`video-url` was proposed by @BreathingFlesh. As we use the keyword `http://` for urls we may go for `http://video` resp. `https://video`?

Reply:
Ok, will go for `video-url` and finalize on Thursday